content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,313 @@
+---
```

Comment:
The path on this doc should not include `istiod`, it should be `.../external-control-plane`.

Reply:
Updated